RecoEgamma/EgammaIsolationAlgos/python/egmIsolationDefinitions_cff.py

```diff
@@ -9,9 +9,8 @@
 pfNoPileUpCandidates = pfAllChargedHadrons.clone()
 pfNoPileUpCandidates.pdgId.extend(pfAllNeutralHadronsAndPhotons.pdgId)
 
-egmIsolationSequence = cms.Sequence( pfParticleSelectionSequence + 
-                                     pfNoPileUpCandidates + 
-                                     egmGedGsfElectronPFNoPileUpIsolation +
-                                     egmGedGsfElectronPFPileUpIsolation +
-                                     pfClusterIsolationSequence
-                                     )
+egmIsolationTask = cms.Task( pfNoPileUpCandidates,
+                                     egmGedGsfElectronPFNoPileUpIsolation,
+                                     egmGedGsfElectronPFPileUpIsolation )
+egmIsolationSequence = cms.Sequence(egmIsolationTask)
+egmIsolationSequence += pfParticleSelectionSequence + pfClusterIsolationSequence
```

Comment:
There is already a pfParticleSelectionTask and you could make a pfClusterIsolationTask
and use them here instead of adding to the sequence.